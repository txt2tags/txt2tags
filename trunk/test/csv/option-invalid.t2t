
Line 1.
%!csv: -z
Line 2.

