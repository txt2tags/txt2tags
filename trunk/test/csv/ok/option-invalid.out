txt2tags: Error: option -z not recognized in %!CSV command
